src/shared/components/home/login-reset.tsx

```diff
@@ -0,0 +1,147 @@
+import { Component, linkEvent } from "inferno";
+import { GetSiteResponse } from "lemmy-js-client";
+import { i18n } from "../../i18next";
+import { UserService } from "../../services";
+import { HttpService } from "../../services/HttpService";
+import {
+  capitalizeFirstLetter,
+  isBrowser,
+  setIsoData,
+  toast,
+  validEmail,
+} from "../../utils";
+import { HtmlTags } from "../common/html-tags";
+import { Spinner } from "../common/icon";
+
+interface State {
+  form: {
+    email: string;
+  };
+  loading: boolean;
+  siteRes: GetSiteResponse;
+}
+
+export class LoginReset extends Component<any, State> {
+  private isoData = setIsoData(this.context);
+
+  state: State = {
+    form: {
+      email: "",
+    },
+    loading: false,
+    siteRes: this.isoData.site_res,
+  };
+
+  constructor(props: any, context: any) {
+    super(props, context);
+  }
+
+  componentDidMount() {
+    if (UserService.Instance.myUserInfo) {
+      this.context.router.history.push("/");
+    }
+  }
+
+  get documentTitle(): string {
+    return `${capitalizeFirstLetter(i18n.t("forgot_password"))} - ${
+      this.state.siteRes.site_view.site.name
+    }`;
+  }
+
+  get isLemmyMl(): boolean {
+    return isBrowser() && window.location.hostname == "lemmy.ml";
+  }
+
+  render() {
+    return (
+      <div className="container-lg">
+        <HtmlTags
+          title={this.documentTitle}
+          path={this.context.router.route.match.url}
+        />
+        <div className="row">
+          <div className="col-12 col-lg-6 offset-lg-3">
+            {this.loginResetForm()}
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  loginResetForm() {
+    return (
+      <div>
+        <form onSubmit={linkEvent(this, this.handlePasswordReset)}>
+          <h5>{capitalizeFirstLetter(i18n.t("forgot_password"))}</h5>
+
+          <div className="form-group row">
+            <label className="col-form-label col-sm-10">
+              {i18n.t("no_password_reset")}
+            </label>
+          </div>
+
+          <div className="form-group row">
+            <label
```

Comment:
What would this label look like if it was a child of the same div as the input its for?

Reply:
As long as it follows the bootstrap conventions, it should be okay.

Reply:
It would look like this @SleeplessOne1917 

<img width="1149" alt="Screenshot 2023-06-20 at 3 20 42 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/35377827/e8123817-6954-40d4-bb96-f0a747cc54c0">